src/target/stm32h7.c

```diff
@@ -473,15 +485,22 @@ static bool stm32h7_uid(target_s *target, int argc, const char **argv)
 	(void)argc;
 	(void)argv;
 
+	struct stm32h7_uid uid = {0};
```

Comment:
This likewise wants moving down to the memcpy and the `= {0}` optionally dropped same as the stm32mp15.c code